database_cleanup/views/purge_properties.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="purge_property_view" model="ir.ui.view">
+        <field name="model">cleanup.purge.wizard.property</field>
+        <field name="inherit_id" ref="form_purge_wizard" />
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <data />
+        </field>
+    </record>
+
+    <record id="action_purge_property" model="ir.actions.server">
+        <field name="name">Purge properties</field>
+        <field name="type">ir.actions.server</field>
+        <field name="state">code</field>
+        <field name="model_id" ref="database_cleanup.model_cleanup_purge_wizard_property" />
+        <field name="code">action = self.get_wizard_action(cr, uid, context=context)</field>
```

Comment:
I get spurious errors
```
ValueError: <type 'exceptions.NameError'>: "name 'self' is not defined" while evaluating u'action = self.get_wizard_action(cr, uid, context=context)'
```

when clicking on some of the menu items. It seems to help to replace the code contents of lines like this with

```action = env['cleanup.purge.wizard.property'].get_wizard_action()```